.ci/script.sh

```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env bash
 
 CI_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+# shellcheck source=/dev/null
 source "${CI_DIR}/common.sh"
 
 travis_retry make fetchthirdparty
+find . -type f -name '*.sh' -not -path "./base/*" -not -path "./luajit-rocks/*" -print0 | xargs --null shellcheck
+find . -type f -name '*.sh' -not -path "./base/*" -not -path "./luajit-rocks/*" -print0 | xargs shfmt -i 0 -w
 make all
 make testfront
-luajit $(which luacheck) --no-color -q {reader,setupkoenv,datastorage}.lua frontend plugins
+luajit "$(which luacheck)" --no-color -q {reader,setupkoenv,datastorage}.lua frontend plugins
```

Comment:
Actually now that I think about it is there a reason to stick this after make all? Shouldn't this also go directly after `make fetchparty`?